src/web_app/js/appcontroller.js

```diff
@@ -454,9 +454,13 @@ AppController.prototype.toggleVideoMute_ = function() {
 AppController.prototype.toggleFullScreen_ = function() {
   if (isFullScreen()) {
     trace('Exiting fullscreen.');
+    document.querySelector('#fullscreen').childNodes[1].innerHTML =
+        '<title>Enter fullscreen</title>';
```

Comment:
Could be simpler like this:

    document.querySelector('svg#fullscreen title').textContent = 'foo';